shopping-cart/shopping-cart-scala/shopping-cart/src/main/scala/com/example/shoppingcart/impl/ShoppingCart.scala

```diff
@@ -35,17 +35,17 @@ object ShoppingCart {
   // Note the "$".
   trait CommandSerializable
 
-  sealed trait Command extends CommandSerializable
+  sealed trait ShoppingCartCommand extends CommandSerializable
```

Comment:
Sounds redundant since this `Command` is inside `ShoppingCart` object. Also, it will be kind of inconsistent with command names below (why not `AddShoppingCartItem` then?). Finally, see [Akka style guide](https://doc.akka.io/docs/akka/2.6/typed/persistence-style.html) which is not using this repetition.

Reply:
Well, I changed it because of a previous comment on this other PR: 
https://github.com/lagom/lagom/pull/2435#discussion_r347494951

If we have a generic `Command`, users need an extra brain cycle. Using `ShoppingCart.Command` is nearly the same as `ShoppingCartCommand`. It's only one `.` shorter. 

In the docs, it was useful to define the commands outside the companion object, because then we can use the whole companion object as a snippet in the docs, without bringing in too much noise. 

This is what we have done here for instance:
https://github.com/lagom/lagom/blob/1.6.x/docs/manual/scala/guide/cluster/code/docs/home/scaladsl/persistence/ShoppingCart.scala#L141-L158